tests/integration/test_category_destpath.py

```diff
@@ -0,0 +1,83 @@
+"""Test if section indexes avoid pages."""
```

Comment:
I think the docstring should be rewritten since it does not test the obsolete _sections_ feature.